main.py

```diff
@@ -375,6 +375,7 @@ def index(marker=None, message=None):
     context['pref_radius'] = pref_radius
     today = datetime.date.today()
     context['default_end_date_format'] = today.strftime('%Y-%m-%d')
+    context['default_start_date_format'] = (today - datetime.timedelta(days=700)).strftime('%Y-%m-%d')
```

Comment:
Actually, only because there is no timedetla for years. 
I talked to Gal and he said that 700 days is o.k.

What do you propose instead?